core/src/main/scala/cats/instances/either.scala

```diff
@@ -80,6 +97,20 @@ trait EitherInstances extends EitherInstances1 {
         b => s"Right(${B.show(b)})"
       )
     }
+
+  implicit def catsDataMonoidForEither[A, B](implicit B: Monoid[B]): Monoid[Either[A, B]] =
+    new Monoid[Either[A, B]] {
+      def empty: Either[A, B] = Right(B.empty)
+      def combine(x: Either[A, B], y: Either[A, B]): Either[A, B] = x combine y
+    }
+
+  implicit def catsDataSemigroupKForEither[L]: SemigroupK[Either[L, ?]] =
+    new SemigroupK[Either[L, ?]] {
+      def combineK[A](x: Either[L, A], y: Either[L, A]): Either[L, A] = x match {
```

Comment:
Yeah, all of the new `either` syntax is copied over from the `Xor` data type. You can see the equivalent here: https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/Xor.scala#L229

That change was made in this PR I believe: https://github.com/typelevel/cats/pull/996